### PR TITLE
Add summary of guidelines for help-wanted issues to help plugin config

### PR DESCRIPTION
**Pull request by MadhavJivrajani**
Add summary of guidelines in `plugins.yaml` as a follow up to https://github.com/kubernetes/test-infra/pull/23140
/sig contributor-experience

### CODE REVIEW

**Comment by MadhavJivrajani** [comment]
/assign @nikhita
for contribex perspective

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -102,6 +102,14 @@ approve:
   - kubernetes-sigs/gateway-api
   lgtm_acts_as_approve: false
 
+help:
+  help_guidelines_summary: |
+                        Please ensure that the issue body includes answers to the following questions:
+                        - Why are we solving this issue?
+                        - If any, what changes need to be made to the code and what places can the assignee treat as reference points?
```
Comment (nimbinatus) [inline config/prow/plugins.yaml:109]: I think folks who have issues with reading comprehension in English, such as folks who don't know English as their first language or people with dyslexia, might have problems with the sentence structure here. Here's a suggestion:

```suggestion
                        - To address this issue, are there any code changes? If there are code changes, what needs to be done in the code and what places can the assignee treat as reference points?
```

**Review by nimbinatus — COMMENTED** [review]
Looks great! Just one suggestion, but it's not a blocker.

File: config/prow/plugins.yaml (hunk 1 above)
Comment (MadhavJivrajani) [reply config/prow/plugins.yaml:109]: This is excellent, thanks a bunch @nimbinatus :)

Comment (MadhavJivrajani) [reply config/prow/plugins.yaml]: Done!

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Comment by MadhavJivrajani** [comment]
/hold cancel
thanks all!

**Pull request merged**